x/bank/client/rest/query_test.go

```diff
@@ -130,6 +133,68 @@ func (s *IntegrationTestSuite) TestTotalSupplyHandlerFn() {
 	}
 }
 
+// TODO move these Gateway tests into a separate file
+func (s *IntegrationTestSuite) TestTotalSupplyGRPCHandler() {
+	val := s.network.Validators[0]
+	baseURL := val.APIAddress
+
+	testCases := []struct {
+		name     string
+		url      string
+		headers  map[string]string
+		respType fmt.Stringer
+		expected fmt.Stringer
+	}{
+		{
+			"test GRPC total supply",
+			fmt.Sprintf("%s/cosmos/bank/v1beta1/supply?height=1", baseURL),
+			map[string]string{
+				grpctypes.GRPCBlockHeightHeader: "1",
+			},
+			&types.QueryTotalSupplyResponse{},
+			&types.QueryTotalSupplyResponse{
+				Supply: sdk.NewCoins(
+					sdk.NewCoin(fmt.Sprintf("%stoken", val.Moniker), s.cfg.AccountTokens),
+					sdk.NewCoin(s.cfg.BondDenom, s.cfg.StakingTokens.Add(sdk.NewInt(10))),
```

Comment:
Tests are failing for any other value (other than `10`) here. They are failing on master too, so we can unblock this and address that via a separate PR